pyvo/conftest.py

```diff
@@ -15,6 +16,8 @@
 except ImportError:
     ASTROPY_HEADER = False
 
+# Keep this until we require numpy to be >=2.0
+np.set_printoptions(legacy="1.25")
```

Comment:
Do you need a numpy version check before calling it?

Reply:
CI says that apparently I do 😅